src/libcore/num/mod.rs

```diff
@@ -1147,6 +1295,13 @@ macro_rules! uint_impl {
         /// negative minimal value for the type); this is a positive
         /// value that is too large to represent in the type. In such
         /// a case, this function returns `MIN` itself.
+        ///
+        /// # Examples
+        ///
+        /// ```rust
+        /// assert_eq!(100i8.wrapping_neg(), -100);
```

Comment:
Travis is showing this as failing: 

	<anon>:2:16: 2:20 error: invalid suffix `i` for numeric literal
<anon>:2     assert_eq!(100i 8.wrapping_neg(), -100);
                        ^~~~
<anon>:2:16: 2:20 help: the suffix must be one of the integral types (`u32`, `isize`, etc)
<anon>:2:21: 2:22 error: no rules expected the token `8`
<anon>:2     assert_eq!(100i 8.wrapping_neg(), -100);
                             ^

I'm not sure why. Looks like maybe you need parens? Seems... odd.